### ISSUE TITLE
Wait 85 seconds to login

### ISSUE THREAD

**Issue by j2l** [saltcorn/saltcorn#1478]
**Describe the bug**
Once you really harden the security of your docker server, the login and registration takes forever (85 seconds) to simply authenticate or register a user.
It also happen when, as admin, you click on About > System, and Plugins.

Container error:
```
fetch store error FetchError: request to https://store.saltcorn.com/api/packs?fields=name,description failed, reason: getaddrinfo EAI_AGAIN store.saltcorn.com
    at ClientRequest.<anonymous> (/usr/local/lib/node_modules/@saltcorn/cli/node_modules/node-fetch/lib/index.js:1461:11)
    at ClientRequest.emit (node:events:390:28)
    at ClientRequest.emit (node:domain:475:12)
    at TLSSocket.socketErrorListener (node:_http_client:447:9)
    at TLSSocket.emit (node:events:390:28)
    at TLSSocket.emit (node:domain:475:12)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  type: 'system',
  errno: 'EAI_AGAIN',
  code: 'EAI_AGAIN'
```
My server has limited access to DNS and external websites on purpose. It shouldn't need to check for stuff on saltcorn.com each time a user authenticate. It looks like telemetry.

**Additional context**
0.7.4-beta.0, may be it's already fixed.

**Comment by glutamate** [comment]
There is definitely no telemetry, but you are right that it is trying to refresh the list of available packs as this is displayed in one of the tabs on the administrators "welcome page" (which is displayed to the administrator if they have not chosen a start page for their role). You can change the URL for the store but of course realistically you are only going to get to that once you have gone through the "admin welcome page"

I agree we should do something about this. The simplest would probably be to replace the list of available packs with a link to `/plugins?set=packs`. Then we should make it easier to disable the module store by setting store endpoint to blank

**Comment by j2l** [comment]
> I agree we should do something about this. The simplest would probably be to replace the list of available packs with a link to /plugins?set=packs. Then we should make it easier to disable the module store by setting store endpoint to blank

Thank you @glutamate, I agree.
Once the webservice is ready for production, store shouldn't need to update except if we visit the plugin section.
You provided a solution so feel free to close this one.

**Comment by glutamate** [comment]
Fixed the packs fetch. the other problem is that it tries to see if an update is available for saltcorn in npm

**Comment by glutamate** [comment]
also fixed in #1487. It will still make the request to npm but only wait 500ms.

**Comment by j2l** [comment]
Thank you @glutamate!
